requirements/pip.in

```diff
@@ -0,0 +1,5 @@
+# Core dependencies for installing other packages
+
+pip==20.0.2
```

Comment:
Why don't we use the latest pip version instead?

Reply:
This version was being explicitly installed in travis config file previously. Using latest version raises errors.